.changeset/strong-rules-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@qwik.dev/core': patch
+---
+
+fix(core): prevent server-side usage of browser APIs
```

Comment:
it's not preventing, it's explaining the error. So more like `FEAT: When an error occurs during SSR due to using the browser APIs, show an explanation.`